qa/truffle/.env

```diff
@@ -0,0 +1,3 @@
+SIDECHAIN_URL=http://localhost:8628/ethv1
+PRIVATE_KEY=a3192defd9ba89e33cacf6ee5bf354006637eedeb9248d36a490c9cd9f6853e7
```

Comment:
We should remove this file, put it in `.gitignore` and add an example file `.env.example`.